mrtt-ui/package.json

```diff
@@ -6,33 +6,45 @@
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
     "@hookform/resolvers": "^2.8.8",
+    "@mapbox/mapbox-gl-draw": "^1.3.0",
     "@mui/icons-material": "^5.6.2",
     "@mui/material": "^5.6.4",
     "@mui/styled-engine-sc": "^5.6.1",
     "@mui/x-date-pickers": "^5.0.0-alpha.2",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.1.1",
     "@testing-library/user-event": "^13.5.0",
+    "@tmcw/togeojson": "^5.1.3",
+    "@turf/area": "^6.5.0",
+    "@turf/bbox": "^6.5.0",
+    "@turf/bbox-polygon": "^6.5.0",
+    "@turf/convex": "^6.5.0",
+    "@types/jest": "^27.4.1",
     "@types/node": "^16.11.27",
     "@types/react": "^18.0.5",
     "@types/react-dom": "^18.0.0",
     "axios": "^0.27.2",
     "date-fns": "^2.28.0",
+    "geojson-prop-types": "^0.2.0",
+    "mapbox-gl": "^2.8.2",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-hook-form": "^7.30.0",
+    "react-map-gl": "^7.0.11",
+    "react-mui-dropzone": "^4.0.6",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "react-toastify": "^9.0.1",
     "sass": "^1.51.0",
+    "shpjs": "^4.0.2",
     "styled-components": "^5.3.5",
     "web-vitals": "^2.1.4",
     "yup": "^0.32.11"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test  --passWithNoTests",
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test  --passWithNoTests",
```

Comment:
Related to this comment: https://github.com/globalmangrovewatch/gmw-users/pull/86/files#r886231154